README.md

```diff
@@ -47,6 +47,18 @@ settings:
   # If label on the issue is not in specified list, this issue will be created as a Bug
   label_mapping:
     enhancement: Story
+
+  # (Optional) JIRA issue's summary customization
+  # Allow to define the JIRA issue's summary and eventually format it with the values
+  # of the GH issue variable.
+  # See GH issue definition : https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28
+  #
+  # Examples:
+  # to use fixed title "github issue": "github issue"
+  # to use GH issue title: "{issue.title}"
+  # to add prefix "[GitHub]" : "[GitHub] {issue.title}"
+  # to add user in the title: "[issue.user.login] {issue.title}"
```

Comment:
will it really work like this ?

I suppose to add the user, you also need to put it into braces `{}`